charts/celestia-node/values.yaml

```diff
@@ -74,6 +74,12 @@ diagnosticMode:
 ## %%MAIN_CONTAINER/POD_DESCRIPTION%%
 ##
 node:
+  ## Select from: bridge, full, light
+  ## Valid values:
+  ## - bridge
+  ## - full
+  ## - light:
+  nodeType: bridge
```

Comment:
Can we move this to `node.settings.nodeType` as we have all the other values to specify what we run?